Bottom: dac637f4a6cb510cc827a0ad750a20ef0fbc8b8f
Top:    eeb2eb6a22664647e2d8a7074ecb07c3de769f8e
Author: Ricardo Salveti <ricardo.salveti@linaro.org>
Date:   2017-02-14 15:28:44 -0200

From 2513de67e7cf488f71c6f8d343b748f476554f60 Mon Sep 17 00:00:00 2001
Subject: [PATCH 18/21] boards: 96b_carbon_nrf51: use pin 29 as UART TX

Current SPI Bluetooth protocol only uses 5 wires, so just reuse the
remaining pin as UART TX (easier to solder and to debug, when needed).

Change-Id: Ib17832b54f4d66b21bf75c730f8db88a73bafc65
Signed-off-by: Ricardo Salveti <ricardo.salveti@linaro.org>


---

diff --git a/boards/arm/96b_carbon_nrf51/96b_carbon_nrf51_defconfig b/boards/arm/96b_carbon_nrf51/96b_carbon_nrf51_defconfig
index e71e031d6..aaa165a29 100644
--- a/boards/arm/96b_carbon_nrf51/96b_carbon_nrf51_defconfig
+++ b/boards/arm/96b_carbon_nrf51/96b_carbon_nrf51_defconfig
@@ -4,11 +4,10 @@ CONFIG_SOC_SERIES_NRF51X=y
 CONFIG_SOC_NRF51822_QFAC=y
 CONFIG_BOARD_96B_CARBON_NRF51=y
 
-# TODO: Disable console/uart
 # enable uart driver
 CONFIG_SERIAL=y
 CONFIG_UART_NRF5=y
-CONFIG_UART_NRF5_GPIO_TX_PIN=9
+CONFIG_UART_NRF5_GPIO_TX_PIN=29
 CONFIG_UART_NRF5_GPIO_RX_PIN=11
 
 # enable console
